qcodes/instrument_drivers/Minicircuits/USB_SPDT.py

```diff
@@ -1,7 +1,12 @@
 from qcodes import Instrument
 from qcodes.utils import validators as vals
 from qcodes.instrument.channel import InstrumentChannel, ChannelList
-
+try:
+    import clr
+except ImportError:
+    raise ImportError("""Module clr not found. Please obtain it by
+                         running 'pip install -i https://pypi.anaconda.org/pythonnet/simple pythonnet' 
```

Comment:
why this extra index. The package seems to be on pypi?

Reply:
And if this is only going to work on windows we probably should check for that first

Reply:
extra index? I am not too familiar with installing python packages so I have just put what worked for me.

Reply:
I dont know if this is a windows only package...

Reply:
` -i https://pypi.anaconda.org/pythonnet/simple` means instead of looking in the normal python packages index go and look in this specific location. That is very unusual and I would not recommend doing anything like that unless you absolutely have too. The package is here https://pypi.python.org/pypi/pythonnet/2.3.0 so doing `pip install pythonnet` should work too. 

The dll is probably windows only so this is unlikely to work on any other platform

Reply:
thanks. that does indeed seem nicer if it works.